CHANGELOG.md

```diff
@@ -4,12 +4,19 @@
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.13.3...master)
 
+## General
+
+## What's New
+
+- A new component was added for customizing how our rust logging is handled. It allows android code to get a callback whenever a log is emitted from Rust (Note: a helper that hooks this directly into the android-components Log system is [forthcoming](https://github.com/mozilla-mobile/android-components/pull/1765)). ([#472](https://github.com/mozilla/application-services/pull/472))
```

Comment:
It's the case that the new release will not log anything until each megazord-consuming application takes an action, right?  (Either by using the a-c bits or by doing some manual plumbing.)

If I'm correct, then I think we should put this in the "Breaking changes" section (per Matt's release notes doc).  If we do move it, let's capitalize Rust and Android consistently throughout.